### ISSUE TITLE
Full Optimized – High Speed

### ISSUE THREAD

**Issue by rusefillc** [rusefi/rusefi#4818]
We shall support "Full Optimized – High Speed" TS mode

**Comment by mck1117** [comment]
What is full optimized mode? I looked in to all the modes enumerated in the PDF and if you're logging lots of fields (like we are), the mode we're using is most efficient.

**Comment by rusefillc** [comment]
``11.2.3 Full Optimized – High Speed`` yes only helpful while focusing on a small subset

**Comment by mck1117** [comment]
What scenario do we think this gives an advantage?

**Comment by mck1117** [comment]
Remember that we're unlikely to exceed 1khz over USB due to protocol limitations.

**Comment by rusefillc** [comment]
I believe @dron0gus wants to explore it for Bluetooth for primary and/or wideband

**Comment by rusefillc** [comment]
Gauges refresh rate more than doubled with java protocol simulator TcpServerSandbox.java and TCP/IP connector


java protocol simulator, TCP/IP connector, without scatteredGetEnabled

![image](https://user-images.githubusercontent.com/48498823/203689198-f147c433-dd34-4406-a85b-7f62669e5f58.png)

java protocol simulator, TCP/IP connector, with scatteredGetEnabled

![image](https://user-images.githubusercontent.com/48498823/203689101-f07c1a7c-4fbd-4765-821b-5481ef00cd44.png)

**Comment by rusefillc** [comment]
@dron0gus current size is 22Kish and protocol limitation for offsets is 8K :(

**Comment by mck1117** [comment]
> @dron0gus current size is 22Kish and protocol limitation for offsets is 8K :(

Aren’t those offsets in to the output channels though, which are only ~1k?

**Comment by rusefillc** [comment]
Oooops, my mistake

**Comment by dron0gus** [comment]
![Screenshot from 2022-11-29 00-40-47](https://user-images.githubusercontent.com/28624689/204386867-40fe5299-b55c-4ef1-b3ee-927d8b6193b1.png)
AlphaX over JDY33.
Should Console@USB and TS@BT work simultaneously?

**Comment by rusefillc** [comment]
> Should Console@USB and TS@BT work simultaneously?

yes absolutely, I would expect any combination of connectors to work in parallel
